includes/GlobalFunctions.php

```diff
@@ -4084,3 +4084,16 @@ function wfRandomString( $length = 32 ) {
 	}
 	return substr( $str, 0, $length );
 }
+
+/**
+ * @param string $url the url to convert to protocol relative
+ * @return string
+ */
+function wfProtocolUrlToRelative( $url ) {
+	$pos = stripos( $url, "://" );
```

Comment:
Minor: Should use single quotes per conventions.

Reply:
fix: 6d46c8630b4